docs/source/venv-update.rst

```diff
@@ -0,0 +1,49 @@
+.. _venv-update-details:
+
+``venv-update`` in detail
+=========================
+
+venv-update is a small script whose job is to idempotently ensure the existence
+of a project's virtualenv based on a set of requirements files.
```

Comment:
it also ensures that the project's virtualenv is in sync with the requirements files